hack/libe2e.sh

```diff
@@ -101,3 +99,49 @@ function stdout_gt() {
     fi
     return 1
 }
+
+function dump_debug_logs() {
+    local namespace="${1}"
+    if [ "${namespace}" != "" ]; then
+        local output_dir="$(pwd)/_artifacts/${namespace}/dump.txt"
+        echo "Dumping cluster state to ${output_dir}"
+        mkdir -p "${output_dir}"
+        kubectl cluster-info dump --namespaces "${namespace}" > "${output_dir}/dump.txt" || true
```

Comment:
You can do `kubectl cluster-info dump --namespaces default,kube-system --output-directory=/path/to/cluster-state` which will output separate yaml and log files.